sdk/contracts/index.ts

```diff
@@ -0,0 +1,134 @@
+import { NetworkId, NetworkNameById } from '@synthetixio/contracts-interface';
+import { Contract } from 'ethers';
+
+import {
+	ExchangerABI,
+	SystemStatusABI,
+	ExchangeRatesABI,
+	SynthUtilABI,
+	SystemSettingsABI,
+	SynthRedeemerABI,
+	FuturesMarketDataABI,
+	FuturesMarketSettingsABI,
+	FuturesMarketABI,
+	SynthABI,
+} from './abis/main';
+
+export type ContractName =
+	| 'Exchanger'
+	| 'SystemStatus'
+	| 'ExchangeRates'
+	| 'SynthUtil'
+	| 'SystemSettings'
+	| 'SynthRedeemer'
+	| 'FuturesMarketData'
+	| 'FuturesMarketSettings'
+	| 'FuturesMarket'
+	| 'Synth';
+
+export type ContractDetails = {
+	name: ContractName;
+	networks: Partial<Record<NetworkId, { address: string; contract: Contract }>>;
+};
+
+export type AllContractsMap = Record<ContractName, Partial<Record<NetworkId, Contract>>>;
+
+export type ContractMap = Partial<Record<ContractName, Contract>>;
```

Comment:
Will be important the contract map includes the specific types for each one once the typechain contracts are implemented as commented above.

Reply:
For sure. This will be added once we ascertain that everything works, and we start cleaning stuff up and adding DX improvements.